### PR TITLE
Tune NPM timeouts and behavior for http2

### PR DESCRIPTION
**Pull request by calebbrown**
#337 

Tweak the behavior of NPM to work better with HTTP2 to reduce timeouts.

### CODE REVIEW

File: pkg/feeds/npm/npm.go [hunk 1]
```diff
@@ -176,70 +179,35 @@ func fetchAllPackages(registryURL string) ([]*feeds.Package, []error) {
...
+			// without being recorded in the json. An `event` could be logged
+			// here.
+			if len(pkgs) > count {
+				packageChannel <- pkgs[:count]
+			} else {
+				packageChannel <- pkgs
+			}
+		}(pkgTitle, count)
+		// Pause briefly between each request so all the requests don't arrive at once.
+		time.Sleep(50 * time.Millisecond)
```
Comment (maxfisher-g) [inline pkg/feeds/npm/npm.go:208]: nit: define the pause duration up in the constants

**Review by maxfisher-g — APPROVED** [review]
Just one suggestion, otherwise LGTM

File: pkg/feeds/npm/npm.go (hunk 1 above)
Comment (calebbrown) [reply pkg/feeds/npm/npm.go]: Woops - I meant to delete that. I've added a random delay above inside the goroutine.

**Review by maxfisher-g — APPROVED** [review]

File: pkg/feeds/npm/npm.go (hunk 1 above)
Comment (maxfisher-g) [reply pkg/feeds/npm/npm.go]: Haha that also solves it

**Pull request merged**